examples/text-generation/README.md

```diff
@@ -212,3 +214,25 @@ python run_generation.py \
 ```
 
 `--bucket_size` option is especially useful when processing an input stream with varying lengths, that is when you have something like `--dataset_name squad --column_name context --max_input_tokens -1`. `--max_input_tokens -1` specifies no truncation of input prompt in the dataset.
+
+### Running with FP8
+llama-7b in FP8 is enabled using these instructions to enable FP8 in synapse:https://docs.habana.ai/en/latest/PyTorch/Inference_on_PyTorch/Inference_Using_FP8.html
+use --kv_cache_fp8 to run with kv cache allocated in fp8
```

Comment:
```suggestion

To use Llama2-7b in FP8, use these instructions to enable FP8 in SynapseAI: https://docs.habana.ai/en/latest/PyTorch/Inference_on_PyTorch/Inference_Using_FP8.html
Add the `--kv_cache_fp8` argument to run the model with a KV cache allocated in fp8.
```

Reply:
with your suggestion, it seems like they need to do the instructions to enable fp8.
but we already done that.. they only need to run with the flag..

will fix this - 
Add the `--kv_cache_fp8` argument to run the model with a KV cache allocated in fp8.

Reply:
I'm still not convinced by the phrasing here. It makes it sound like the user should visit the Gaudi doc to make it work, which is not the case. We should rather have something like: "More information here: [link]".
Also, I would add that `--enable quantization` enables to run matrix multiplications in fp8.

Reply:
fixed.